ui/round/src/plugins/round.nvui.ts

```diff
@@ -199,7 +199,7 @@ export function initModule(): NvuiPlugin {
           : playable(ctrl.data)
             ? renderTablePlay(ctrl)
             : renderTableEnd(ctrl)),
-        h('h2', i18n.settings.settings),
```

Comment:
I didn't include the settings i18nModule as well just for this one i18n. I would be more in favour of transferring something as common as a settings i18n over to site.